playground/backend/internal/environment/application.go

```diff
@@ -90,14 +90,22 @@ type ApplicationEnvs struct {
 
 	// pipelineExecuteTimeout is timeout for code processing
 	pipelineExecuteTimeout time.Duration
+
+	// launchSite is a launch site of application
+	launchSite string
+
+	// googleProjectId is the Google Сloud project id
+	googleProjectId string
 }
 
 // NewApplicationEnvs constructor for ApplicationEnvs
-func NewApplicationEnvs(workingDir string, cacheEnvs *CacheEnvs, pipelineExecuteTimeout time.Duration) *ApplicationEnvs {
+func NewApplicationEnvs(workingDir, launchSite, googleCloudProjectId string, cacheEnvs *CacheEnvs, pipelineExecuteTimeout time.Duration) *ApplicationEnvs {
```

Comment:
googleCloudProjectId -> usually it is just named projectId

Reply:
Done.